gs_csr.json

```diff
@@ -0,0 +1,42 @@
+[
+  [
+    {
+      "Name" : "CSR_Gauss-siedel",
+      "Input Type" : "IEgenLIB Relation",
+      "Relation" : "{ [i,j] -> [ip,jp] : i<ip && i=colidx(jp) && 0 <= i && i < m && 0 <= ip && ip < m && rowptr(i) <= j && j < rowptr(i+1) && rowptr(ip) <= jp && jp < rowptr(ip+1) }",
+      "Expected" : "{ [i] -> [ip,jp] : i=colidx(jp) && colidx(jp) < ip && ip < m-1 && rowptr(i) < rowptr(i+1) && rowptr(ip) <= jp && jp < rowptr(ip+1) && colidx(jp) >= 0 && rowptr(i) >= 0 && rowptr(ip) >= 0 && rowptr(i+1) < nnz && rowptr(ip+1) < nnz}",
+      "UFS" :
+      [
+        { 
+           "Name" : "colidx",
+           "Domain" : "{[i]:0<=i &&i<nnz}",
+           "Range" : "{[j]:0<=j &&j<m}",
+           "Bijective" : "false",
+           "Monotonicity" : "iegenlib::Monotonic_NONE"
+        },
+        { 
+           "Name" : "rowptr",
+           "Domain" : "{[j]:0<=j &&j<m}",
+           "Range" : "{[i]:0<=i &&i<nnz}",
+           "Bijective" : "false",
+           "Monotonicity" : "iegenlib::Monotonic_Increasing"
+        }
+      ],
+      "User Defined" :
+      [
+
+      ],
+      "Do Not Project" :
+      [
+        "0", "2"
```

Comment:
Let's change the "Do Not Project" to "i" and "ip".  Also call it, "Do Not Project Out"  or "Do Not Eliminate".